internal/api/v1/models/app.go

```diff
@@ -83,6 +94,20 @@ func (ar *AppRef) EnvSecret() string {
 	return names.GenerateResourceName(ar.Name + "-env")
 }
 
+// SvcSecret returns the name of the kube secret holding the
+// bound services of the referenced application
+func (ar *AppRef) SvcSecret() string {
```

Comment:
This is a generator, not a getter (shouldn't be used to find the name of an existing secret). Maybe the name of the method should make that clear? (e.g. `GenSvcSecretName`)

Reply:
Could be done. There are several methods like this in the file now (First was for environment, `EnvSecret` nearby).

Reply:
Done, commit 4b318c1c1d975ef6571608271baf99f66951d126